ci-operator/config/openshift/api/openshift-api-master.yaml

```diff
@@ -147,6 +147,14 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-upgrade-aws
+- as: e2e-upgrade-out-of-change
+  run_if_changed: ^Dockerfile\.ocp$|^Makefile$|^(manifests/.*|payload-command/.*|payload-manifests/.*)$
+  steps:
+    cluster_profile: azure4
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:latest
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:initial
+    workflow: openshift-upgrade-azure
```

Comment:
The e2e-upgrade above uses AWS, can we be consistent with that please?

Reply:
it seemed like it might be worth spreading exposure around, just in case there was a change that broke one cloud.  But the benefit of the platform-agnostic job name is that its really easy to move from cloud to cloud for any reason, so sure, pushed a pivot to AWS with bf1ffcb082 -> 780babb39a.